restructured Login portion and added info about web authentication (#290)
path-cc/path-cc.github.io: _facility/registration.md

Once PATh staff receive your email verification, a Research Computing Facilitator will contact you within one business day to arrange a short consultation and introduction to PATh resources. During this meeting, our staff will provide personalized start-up guidance per your specific computational research goals and activate your account.

> During the meeting your account will also be approved. Upon approval, you will receive a confirmation email.

Reviewer: ```suggestion
Following the meeting, the Facilitator will approve your account, add your profile to any relevant PATh 'project' names, and ensure that you have access to the PATh Facility. (You will receive automated emails for some of these actions, which you can otherwise ignore.)
```

## Login

> Once your account has been approved, it is possible to login to the PATh access point, `submit6.chtc.wisc.edu`, using a terminal or SSH program. Logging in requires authenticating your credientials using one of two options: __web authentication__ or __SSH key pair authenticaiton__. Additional information on this process will be provided during your meeting with a Research Computing Facilitator.

Reviewer: ```suggestion
Once your account has been added to a PATh access point, you will be able to log in using a terminal or SSH program. Logging in requires authenticating your credientials using one of two options: __web authentication__ or __SSH key pair authentication__. Additional information on this process will be provided during and/or following your meeting with a Research Computing Facilitator.
```

### Option 1: Login via Web Authentication

Logging in via web authentication requires no preparatory steps beyond having access to an internet browser. 

To authenticate using this approach: 

1. Open a terminal and type `ssh username@submit6.chtc.wisc.edu`, being sure to replace `username` with your PATh access point username. Upon hitting enter, the following text should appear with a unique, but similar, URL: 

```
Authenticate at
-----------------
https://cilogon.org/device/?user_code=FF4-ZX6-9LK
-----------------

> Hit enter when you authenticate

Reviewer: ```suggestion
Type 'Enter' when you authenticate.
```

```

2. Copy the `https://` link, paste it into a web browser, and hit enter.  

3. You will be redirected to a new page where you will be prompted to login using your institutional credentials. Once you have done so, a new page will appear with the following text: "You have successfully approved the user code. Please return to your device for further instructions."

> 4. Return to your terminal, and hit the enter key to complete the login process. 

Reviewer: ```suggestion
4. Return to your terminal, and type 'Enter' to complete the login process. 
```

### Option 2: Login via SSH Key Pair Authentication

> It is also possible to authenticate using an SSH key pair. Logging in using SSH keys does not require access to an internet browser to login into the PATh access point, `submit6.chtc.wisc.edu`. 

Reviewer: ```suggestion
It is also possible to authenticate using an SSH key pair, if you prefer. Logging in using SSH keys does not require access to an internet browser to login into the PATh access point, `submit6.chtc.wisc.edu`. 
```
